src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -2121,4 +2136,52 @@ public void processStanza(Stanza stanza)
             }
         }
     }
+
+    /**
+     * Used to check the jvb side of the call for any activity.
+     */
+    private class MediaActivityChecker
+        extends TimerTask
+    {
+        @Override
+        public void run()
+        {
+            // if the call was stopped before we check ignore
+            if (!started)
+            {
+                return;
+            }
+
+            CallPeer peer = jvbCall.getCallPeers().next();
```

Comment:
Should the cases of `peer == null` or `mediaHandler == null` or `stream == null` also trigger stopping the conference (there is probably *some* problem)?

Reply:
Probably ... this checks came from another code where we do it early in the code and retry ... will change that.